src/libs/cluster-utils.js

```diff
@@ -82,6 +82,17 @@ export const trimClustersOldestFirst = _.flow(
 
 export const currentCluster = _.flow(trimClustersOldestFirst, _.last)
 
+export const trimAppsOldestFirst = _.flow(
```

Comment:
I don't like the fact that this is duplicated from cluster logic, but don't want to rely on them being equivalent forever.

Reply:
In fact the implementation of `trimClustersOldestFirst` has a bug, which is being fixed on the PD branch, so these will end up being different shortly. Even if that wasn't the case, this definitely looks like 'incidental equivalence' as opposed to 'intentional equivalence', so keeping them separate is a good way to communicate that.

Reply:
Ah yes! Indeed!